inc/database.inc.php

```diff
@@ -67,7 +67,6 @@ function dbConnect() {
     global $db_charset;
     global $db_file;
     global $db_debug;
-    global $db_ssl_ca;
```

Comment:
It was supported here but only for mysqli: https://github.com/poweradmin/poweradmin/blob/9731ac8dfe42f67d011058f486f7fb3a9b01d67f/inc/database.inc.php

The possibility also exists with Doctrine DBAL but I don't see a need to implement it as it's very specific. (http://stackoverflow.com/questions/19039947/symfony2-doctrine-connect-to-database-via-ssl)